### PR TITLE
Issue 49 qasm string input

### PR DESCRIPTION
**Pull request by neal-erickson**
@qciaran is this what you were looking for with issue #49?

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -69,3 +69,9 @@ def test_pytket_classical_only(self) -> None:
             "returns": [["a", 0], ["b", 0]],
             "args": [["b", 2], ["a", 1]],
         }
+
+    def test_qasm_to_phir(self) -> None:
+        """Test the qasm string entrypoint works."""
+        qasm = 'OPENQASM 2.0;include "qelib1.inc";qreg q[3];h q; ZZ q[1],q[0];creg cr[3];measure q[0]->cr[0];measure q[1]->cr[0];'  # noqa: E501
```
Comment (qartik) [inline tests/test_api.py:75]: Will be easier to read if it's a multiline string.

Comment (neal-erickson) [reply tests/test_api.py]: Yeah, fair. Updated.

File: pytket/phir/cli.py [hunk 2]
```diff
@@ -41,6 +41,13 @@ def main() -> None:
         default="H1-1",
         help="machine name, H1-1 by default",
     )
+    parser.add_argument(
+        "-t",
```
Comment (qartik) [inline pytket/phir/cli.py:45]: Would it be preferable to use "-O" here as common for optimization levels?

File: pytket/phir/cli.py [hunk 3]
```diff
@@ -52,7 +59,8 @@ def main() -> None:
     for file in args.qasm_files:
         print(f"Processing {file}")  # noqa: T201
         c = circuit_from_qasm(file)
-        rc = rebase_to_qtm_machine(c, args.machine)
+        tket_level = int(args.tk)
```
Comment (qartik) [inline pytket/phir/cli.py:62]:
```suggestion
        tket_opt_level = int(args.tk)
```
or even `opt_level`

**Review by qartik — APPROVED** [review]
Minor changes suggested, but otherwise LGTM.

**Comment by neal-erickson** [comment]
@qartik good suggestions, made those changes

**Review by qartik — APPROVED** [review]

**Comment by qartik** [comment]
> @qciaran is this what you were looking for with issue 49?

we should probably wait for @qciaran before merging this?

**Comment by qciaran** [comment]
giphy
Thanks!

**Review by qciaran — APPROVED** [review]

**Pull request merged**